### PR TITLE
[DLS] Adds GraphQL datastes info to CMB docs

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+| | Magic WAN | US and EU <br> `MagicWANConnectorMetricsAdaptiveGroups` |
+| | Spectrum | US only <br> [`spectrumNetworkAnalyticsAdaptiveGroups`](/analytics/graphql-api/migration-guides/network-analytics-v2/node-reference/) |
+| Platform | GraphQL Analytics API | [All GraphQL Analytics API datasets](/analytics/graphql-api/features/discovery/introspection/) | 
+| | Logpush | US only <br> [`logpushHealthAdaptiveGroups`](/logs/get-started/alerts-and-analytics/#enable-logpush-health-analytics) <br> |
+| Zero Trust | Access | US and EU <br> [`accessLoginRequestsAdaptiveGroups`](/analytics/graphql-api/tutorials/querying-access-login-events/) | |
+| | Browser Isolation | US and EU <br> Only the field `isIsolated` part of `GatewayL7RequestsAdaptiveGroups` dataset |
```
Comment (soheiokamoto) [inline content/data-localization/metadata-boundary/graphql-datasets.md]: To be consistent with other descriptions:
```suggestion
| | Browser Isolation | US and EU <br> Only the field `isIsolated` part of `GatewayL7RequestsAdaptiveGroups`  |
```

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+| | DNS Firewall | Same as DNS |
+| | DMARC Management | US and EU <br> `dmarcReportsAdaptive` <br> `dmarcReportsSourcesAdaptiveGroups` | |
+| | Page Shield | US and EU <br> [`pageShieldReportsAdaptiveGroups`](/page-shield/policies/violations/#get-policy-violations-via-graphql-api) |
+| | SSL | US and EU <br> Only the fields `clientSSLProtocol` and `ja3Hash` part of `httpRequestsAdaptive` and `httpRequestsAdaptiveGroups` |  |
+| | SSL 4 SaaS | US and EU <br> [clientRequestHTTPHost](/cloudflare-for-platforms/cloudflare-for-saas/hostname-analytics/#explore-customer-usage) <br> Refer to [GraphQL Tutorial on querying HTTP events by hostname](/analytics/graphql-api/tutorials/end-customer-analytics/) |
+| | Turnstile | US and EU <br> [`turnstileAdaptiveGroups`](/turnstile/turnstile-analytics/#graphql) |
+| | WAF/L7 Firewall | US and EU <br> [`firewallEventsAdaptive`](/analytics/graphql-api/tutorials/querying-firewall-events/) <br> [`firewallEventsAdaptiveGroups`](https://blog.cloudflare.com/how-we-used-our-new-graphql-api-to-build-firewall-analytics/) <br> [`firewallEventsAdaptiveByTimeGroups`](/logs/reference/change-notices/2023-02-01-security-fields-updates/) |
```
Comment (pedrosousa) [inline content/data-localization/metadata-boundary/graphql-datasets.md]: Not sure if it's worth linking `firewallEventsAdaptiveByTimeGroups` to `/logs/reference/change-notices/2023-02-01-security-fields-updates/`: It doesn't contain relevant information about the dataset, just a couple of changes.

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| | Email Routing | US and EU <br> `emailRoutingAdaptive` <br> `emailRoutingAdaptiveGroups` | | 
+| | R2 | US and EU <br> `r2OperationsAdaptiveGroups` <br> `r2StorageAdaptiveGroups` | | 
+| | Stream | US only <br> [`streamMinutesViewedAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoPlaybackEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoBufferEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoQualityEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) |
+| | Workers (deployed on a Zone) | US and EU <br> `workerPlacementAdaptiveGroups` <br> US only <br> `workersZoneInvocationsAdaptiveGroups` <br> `workersZoneSubrequestsAdaptiveGroups` <br> `workersOverviewRequestsAdaptiveGroups` <br> `workersOverviewDataAdaptiveGroups` <br> [`workersAnalyticsEngineAdaptiveGroups`](/analytics/analytics-engine/get-started/#4-query-data-using-graphql-and-sql-api) <br> [`workersInvocationsAdaptive`](/analytics/graphql-api/tutorials/querying-workers-metrics/) <br> `workersInvocationsScheduled` <br> `workersSubrequestsAdaptiveGroups` |
```
Comment (pedrosousa) [inline content/data-localization/metadata-boundary/graphql-datasets.md]: The anchor "#4-query-data-using-graphql-and-sql-api" no longer exists on the target page.

**Review by pedrosousa — APPROVED** [review]

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+| | Email Routing | US and EU <br> `emailRoutingAdaptive` <br> `emailRoutingAdaptiveGroups` | | 
+| | R2 | US and EU <br> `r2OperationsAdaptiveGroups` <br> `r2StorageAdaptiveGroups` | | 
+| | Stream | US only <br> [`streamMinutesViewedAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoPlaybackEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoBufferEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) <br> [`videoQualityEventsAdaptiveGroups`](/stream/getting-analytics/fetching-bulk-analytics/) |
+| | Workers (deployed on a Zone) | US and EU <br> `workerPlacementAdaptiveGroups` <br> US only <br> `workersZoneInvocationsAdaptiveGroups` <br> `workersZoneSubrequestsAdaptiveGroups` <br> `workersOverviewRequestsAdaptiveGroups` <br> `workersOverviewDataAdaptiveGroups` <br> `workersAnalyticsEngineAdaptiveGroups` <br> [`workersInvocationsAdaptive`](/analytics/graphql-api/tutorials/querying-workers-metrics/) <br> `workersInvocationsScheduled` <br> `workersSubrequestsAdaptiveGroups` |
```
Comment (jimhawkridge) [inline content/data-localization/metadata-boundary/graphql-datasets.md]: `workersAnalyticsEngineAdaptiveGroups` is available in the EU now.

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+| Network Services | Network Error Logging (NEL) / Edge Reachability / Last Mile Insights | US only <br> `nelReportsAdaptiveGroups` |
+| | Magic Firewall |  US only <br> [`magicFirewallSamplesAdaptiveGroups`](/magic-firewall/tutorials/graphql-analytics/) <br> [`magicFirewallNetworkAnalyticsAdaptiveGroup`](/magic-firewall/tutorials/graphql-analytics/#example-queries-for-magic-firewall) |
```
Comment (jimhawkridge) [inline content/data-localization/metadata-boundary/graphql-datasets.md]:
```suggestion
| | Magic Firewall |  US only <br> [`magicFirewallSamplesAdaptiveGroups`](/magic-firewall/tutorials/graphql-analytics/) <br> [`magicFirewallNetworkAnalyticsAdaptiveGroups`](/magic-firewall/tutorials/graphql-analytics/#example-queries-for-magic-firewall) | 
```

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+| | Workers (deployed on a Zone) | US and EU <br> `workerPlacementAdaptiveGroups` <br> US only <br> `workersZoneInvocationsAdaptiveGroups` <br> `workersZoneSubrequestsAdaptiveGroups` <br> `workersOverviewRequestsAdaptiveGroups` <br> `workersOverviewDataAdaptiveGroups` <br> `workersAnalyticsEngineAdaptiveGroups` <br> [`workersInvocationsAdaptive`](/analytics/graphql-api/tutorials/querying-workers-metrics/) <br> `workersInvocationsScheduled` <br> `workersSubrequestsAdaptiveGroups` |
+| Network Services | Network Error Logging (NEL) / Edge Reachability / Last Mile Insights | US only <br> `nelReportsAdaptiveGroups` |
+| | Magic Firewall |  US only <br> [`magicFirewallSamplesAdaptiveGroups`](/magic-firewall/tutorials/graphql-analytics/) <br> [`magicFirewallNetworkAnalyticsAdaptiveGroup`](/magic-firewall/tutorials/graphql-analytics/#example-queries-for-magic-firewall) | 
+| | Magic Network Monitoring | US and EU <br> [`mnmFlowDataAdaptiveGroups`](/magic-network-monitoring/tutorials/graphql-analytics/) |
```
Comment (jimhawkridge) [inline content/data-localization/metadata-boundary/graphql-datasets.md]:
```suggestion
| | Magic Network Monitoring | US only <br> [`mnmFlowDataAdaptiveGroups`](/magic-network-monitoring/tutorials/graphql-analytics/) |
```

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+| | Magic Transit | US only <br> [`magicTransitNetworkAnalyticsAdaptiveGroups`](/analytics/graphql-api/migration-guides/network-analytics-v2/node-reference/) <br> [`flowtrackdNetworkAnalyticsAdaptiveGroups`](/analytics/graphql-api/migration-guides/network-analytics-v2/node-reference/) <br> [`magicTransitTunnelHealthChecksAdaptiveGroups`](/analytics/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/) <br> [`magicTransitTunnelTrafficAdaptiveGroups`](/magic-transit/analytics/query-bandwidth/) | 
+| | Magic WAN | US and EU <br> `MagicWANConnectorMetricsAdaptiveGroups` |
+| | Spectrum | US only <br> [`spectrumNetworkAnalyticsAdaptiveGroups`](/analytics/graphql-api/migration-guides/network-analytics-v2/node-reference/) |
+| Platform | GraphQL Analytics API | [All GraphQL Analytics API datasets](/analytics/graphql-api/features/discovery/introspection/) |
```
Comment (jimhawkridge) [inline content/data-localization/metadata-boundary/graphql-datasets.md]: Do we mean to say `US and EU` here?

File: content/data-localization/metadata-boundary/graphql-datasets.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+| | Magic WAN | US and EU <br> `MagicWANConnectorMetricsAdaptiveGroups` |
+| | Spectrum | US only <br> [`spectrumNetworkAnalyticsAdaptiveGroups`](/analytics/graphql-api/migration-guides/network-analytics-v2/node-reference/) |
+| Platform | GraphQL Analytics API | [All GraphQL Analytics API datasets](/analytics/graphql-api/features/discovery/introspection/) | 
+| | Logpush | US only <br> [`logpushHealthAdaptiveGroups`](/logs/get-started/alerts-and-analytics/#enable-logpush-health-analytics) <br> |
+| Zero Trust | Access | US and EU <br> [`accessLoginRequestsAdaptiveGroups`](/analytics/graphql-api/tutorials/querying-access-login-events/) | |
+| | Browser Isolation | US and EU <br> Only the field `isIsolated` part of `GatewayL7RequestsAdaptiveGroups`  |
```
Comment (jimhawkridge) [inline content/data-localization/metadata-boundary/graphql-datasets.md]:
```suggestion
| | Browser Isolation | US and EU <br> Only the field `isIsolated` part of `gatewayL7RequestsAdaptiveGroups`  |
```

**Review by jimhawkridge — CHANGES_REQUESTED** [review]
I've cross checked the graphql ones against the config and have some suggestions.

**Review by jimhawkridge — APPROVED** [review]

**Pull request merged**